src/ui/server/index.ts

```diff
@@ -44,6 +44,7 @@ const cookie = {
 } as csrf.CookieOptions;
 
 const sessionOptions = {
+  name: '__Secure-exip-session',
```

Comment:
Getting some errors with this running locally

![Screenshot 2023-02-10 at 13 39 36](https://user-images.githubusercontent.com/2463648/218106008-9b7d4814-4f30-4646-aa34-aa246d9ea23a.png)

Reply:
Thanks, it is failing because localhost is not running on HTTPS.
Have updated.